cli/commands/console.go

```diff
@@ -127,6 +130,34 @@ func (o *ConsoleOptions) Run() error {
 	consoleTemplateVersion := o.EC.Version.GetConsoleTemplateVersion()
 	consoleAssetsVersion := o.EC.Version.GetConsoleAssetsVersion()
 
+	// Determine from where assets should be served
+	getConfig := func(url string, target interface{}) error {
```

Comment:
can we have this a method in `cli.ServerConfig` and call this method on `ec.Validate`? https://github.com/hasura/graphql-engine/blob/master/cli/cli.go#L63

Anyway, we are contacting server `v1/version` to get the server version. And `v1alpha1/config` also returns version.

Reply:
Okay, let's use the config api to get all the details. I'd still recommend putting this behind a feature flag -- just to be on the safe side.

Reply:
@shahidhk feature flags can be used only when we have the server version in the CLI.